ledger-core/virtual/object/working_table.go

```diff
@@ -14,28 +14,29 @@ import (
 )
 
 type WorkingTable struct {
-	lists map[contract.InterferenceFlag]*WorkingList
+	requests []*WorkingList
+	results  map[reference.Global]Summary
 }
 
 func NewWorkingTable() WorkingTable {
 	var rt WorkingTable
-	rt.lists = make(map[contract.InterferenceFlag]*WorkingList)
+	rt.requests = make([]*WorkingList, 3)
```

Comment:
3?

Reply:
const (
	_ InterferenceFlag = iota
	CallIntolerable
	CallTolerable

	InterferenceFlagCount = iota
)

Reply:
it cheaper to build it in one go
```
rt.requests{
contract.CallTolerable: NewWorkingList(),
contract.CallIntolerable: NewWorkingList(),
}
```

Reply:
we expect 2 values only

Reply:
I do not want to use arithmetic when using constant as id.

Reply:
not sure what you are doing here rt.requests = make([]*WorkingList, 3)
`requests []*WorkingList`, `rt.requests[contract.CallTolerable] = NewWorkingList()` ??

Reply:
Okay, I will change this.

Reply:
OK So you've switched from map to slice. I don't really like the because 0 is not expected to be valid value here. BUt memorywise it's cheaper.

Reply:
yep, this is optimized for fast search by index

Reply:
@ruz really asked to remove map here)

Reply:
don't use in-code constant here, then, use `InterferenceFlagCount`

Reply:
Fixed